src/repos/WeeklyDebriefRepo.ts

```diff
@@ -0,0 +1,55 @@
+import { DocumentType } from '@typegoose/typegoose';
+import { User, UserModel } from '../entities/User';
+import WeeklyDebrief from '../entities/WeeklyDebrief';
+import { ArticleModel } from '../entities/Article';
+
+const getExpirationDate = (creationDate: Date): Date => {
+  creationDate.setDate(creationDate.getDate() + 7);
+  return creationDate;
+};
+
+const createWeeklyDebrief = async (
+  user: DocumentType<User>,
+  creationDate: Date,
+  expirationDate: Date,
+): Promise<User> => {
+  const articleAggregate = ArticleModel.aggregate();
+  const { uuid } = user;
+  const weeklyDebrief = Object.assign(new WeeklyDebrief(), {
+    uuid,
+    creationDate,
+    expirationDate,
+    numShoutouts: user.numShoutouts,
+    numBookmarkedArticles: user.numBookmarkedArticles,
+    readArticles: user.readArticles.slice(0, 2),
```

Comment:
what if the length is not more than 2.... this would give an error?

Reply:
It would return the entire array
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice